src/python/thrift_examples/person_test.py

```diff
@@ -0,0 +1,6 @@
+from thrift_example.gen.ttypes import Person
```

Comment:
It maybe is worth a comment to explain how this value came about via the `namespace` command? It took me a minute to put it together.

Reply:
Done.